raml-parser-interface/src/main/java/org/mule/apikit/ParserUtils.java

```diff
@@ -85,6 +87,18 @@ public static String queryStringAsYamlValue(Map<String, Parameter> facets,
     return queryStringYaml.toString();
   }
 
+  /**
+   * Check if the query parameters values is considered null. Query parameters are being consider null either if the collection is
+   * null, the collection has a single null value or the collection has a single string value 'null'
+   *
+   * @param paramValues
+   * @return true if the query parameters values is considered null otherwise false
+   */
```

Comment:
```suggestion
  /**
   * Check if the values for the query parameter are considered null. Query parameters are considered null when the whole collection is
   * null, has a single null value or has a single 'null' string value
   *
   * @param paramValues
   * @return true if the values for the query parameter are considered null otherwise false
   */
```